app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ContinuousScanActivity.java

```diff
@@ -463,6 +479,10 @@ public void onSlide(@NonNull View bottomSheet, float slideOffset) {
         });
     }
 
+    private Activity getActivity() {
+        return this.getActivity();
+    }
+
     private boolean isProductIncomplete() {
```

Comment:
Why is this method created? It is not used anywhere and this method recurses infinitely and can only end by throwing an exception. I guess this method should be removed.